pkg/provisioners/heartbeats/config/examples/subscription.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Subscription
+metadata:
+  name: love-dumper
+  namespace: default
+spec:
+  from:
+    kind: Source
+    apiVersion: eventing.knative.dev/v1alpha1
+    name: love
+  call:
+    target:
+      kind: Service
+      apiVersion: serving.knative.dev/v1alpha1
+      name: message-dumper
```

Comment:
is it possible to also use an `image` here ? 

I see you building your `message-dumper` Golang-container using `ko`, but was wondering if `ko` is required for building "sources" ? (and other artifacts in the new spec) ?

Reply:
ko is just a nice way to push containers to a registry if you are using go.